src/main/resources/db/migration/V021__AddStateToCaseTable.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TYPE public.CASE_STATE AS ENUM (
+    'OPEN',
+    'PENDING_CLOSURE',
+    'CLOSED'
+);
+
+ALTER TABLE public.cases
+    ADD COLUMN state CASE_STATE DEFAULT 'OPEN' NOT NULL,
```

Comment:
likewise with above, new fields all need to be optional for this PR to be merged earlier